src/contrasts.jl

```diff
@@ -196,11 +228,22 @@ end
 Coding that generates one indicator (1 or 0) column for each level,
 __including__ the base level.
 
-Needed internally when a term is non-redundant with lower-order terms (e.g., in
-`~0+x` vs. `~1+x`, or in the interactions terms in `~1+x+x&y` vs. `~1+x+y+x&y`. In the
-non-redundant cases, we can expand x into `length(levels(x))` columns
-without creating a non-identifiable model matrix (unless the user has done
-something foolish in specifying the model, which we can't do much about anyway).
+Not exported but included here for the sake of completeness.
```

Comment:
We do export them. They're the user-facing interface for specifying contrast coding strategy.